### DISCUSSION TITLE
Requesting help with code integration

### DISCUSSION THREAD

**Discussion by Aidanmathias** [custom-components/pyscript#476] [Q&A] [answered]
Hello and good day.

I am working on a website that needs to have a dropdown selector to change the degrees from Celsius to Fahrenheit and I'm having trouble integrating it with PyScript. I have attached an image of a mockup for the code minus exporting the result to a table. Is it possible that anyone could help me find a solution or at least an alternative to the issue?

![Screenshot 2023-06-21 135218](https://github.com/custom-components/pyscript/assets/135874568/be837ac8-9a1a-4eb0-b2bd-715c42f5800d)

**Comment by craigbarratt** [comment, marked as answer]
You have the wrong `pyscript`.  This `pyscript` is a scripting language extension to HASS.
